modules/modelSampler/StableDiffusionVaeSampler.py

```diff
@@ -38,24 +38,31 @@ def sample(
         # TODO: this is reusing the prompt parameters as the image path, think of a better solution
         image = Image.open(sample_config.prompt)
         image = image.convert("RGB")
-
-        t_in = transforms.ToTensor()
+        # TODO: figure out better set of transformations for resize and/or implement way to configure them as per-sample toggle
+        scale = sample_config.height
+        if sample_config.width > sample_config.height:
+            sample_config.width
```

Comment:
this line doesn't do anything useful

Reply:
Yeah, it should be `scale = sample_config.width` after if. To make sure resize is done with longer dimension. I'll fix it before weekend.